motors/tacho_motor_class.c

```diff
@@ -342,6 +343,7 @@ static int tm_do_one_ramp_step(struct tacho_motor_device *tm,
 		if (params->command == TM_COMMAND_STOP)
 			return tm->ops->stop(tm->context,
 					     params->stop_action);
+		tacho_motor_notify_state_change(tm);
```

Comment:
This is placed here because `tm->ops->stop(tm->context, params->stop_action))` calls `sysfs_notify`.